lib/cirro_io_v2/request_clients/jwt.rb

```diff
@@ -18,7 +18,8 @@ def initialize(base_url:, private_key:, client_id:)
         end
       end
 
-      def make_request(http_method, url, body = nil, params = nil, _headers = {})
+      def make_request(http_method, url, body: nil, params: nil, _headers: {})
+        # TODO: why do you use underscore here?
```

Comment:
You mean `_headers`? it must have come from the linter. It's not used in the method.
I suppose it should be added to the request headers:

```
request.headers = headers.merge('Authorization' => bearer_token)
```